migration/migrator/migrations/course/20200608034200_peer_feedback.py

```diff
@@ -0,0 +1,105 @@
+"""Migration for a given Submitty course database."""
+
+
+def up(config, database, semester, course):
+    """
+    Run up migration.
+
+    :param config: Object holding configuration details about Submitty
+    :type config: migrator.config.Config
+    :param database: Object for interacting with given database for environment
+    :type database: migrator.db.Database
+    :param semester: Semester of the course being migrated
+    :type semester: str
+    :param course: Code of course being migrated
+    :type course: str
+    """
+
+    # Create overall comment table
+    database.execute(
+        """
+       CREATE TABLE IF NOT EXISTS peer_feedback (
+            uid integer NOT NULL,
```

Comment:
What is `uid` here?

Reply:
I think we tend to name unique id's based on the table. For example, this would be `pf_id` for "peer feedback id"

Reply:
uid has been changed to pf_id